cmd/kritis/kubectl/plugins/resolve/cmd/root.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/grafeas/kritis/pkg/kritis/kubectl/plugins/resolve"
+	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
+)
+
+const (
+	PWD                                 = "PWD"
+	KUBECTL_PLUGINS_LOCAL_FLAG_FILENAME = "KUBECTL_PLUGINS_LOCAL_FLAG_FILENAME"
+)
+
+var (
+	files multiArg
+)
+
+func init() {
+	RootCmd.PersistentFlags().VarP(&files, "filename", "f", "Filename to resolve. Set it repeatedly for multiple filenames.")
+}
+
+var RootCmd = &cobra.Command{
+	Use: "resolve-tags",
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		return resolveFilepaths()
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := resolve.Execute(files, os.Stdout); err != nil {
```

Comment:
instead of os.Stdout you can use, cmd.OutOrStdout().
This will helpful in testing the RootCmd.
You can add Tests for  end to end tests for RootCmd, 
```
	var OutputBuffer bytes.Buffer
	RootCmd.SetOutput(&OutputBuffer)
	RootCmd.SetArgs([]string{"--filename=<some temp file with temp data>"})
```